vista_2d/vista_2d_tutorial_monai.ipynb

```diff
@@ -0,0 +1,1405 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #99.    num_epochs = 10
</pre><p>Can we add some descriptions about which hyper-parameters can be adjusted to achieve better performance? According to the figures, training does not converge</p><p>Like using all dataset? using more epochs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Sure, let me add some more description there.</p>

Reply:
I've added more description.